web/src/app/projects/tokens/tokens.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+import {  Observable } from 'rxjs';
+import {  map, take } from 'rxjs/operators';
+
+// Dashboard model and services
+import { TokenService } from '../../core/services/token.service';
+import { ProjectTokenModel } from '../../shared/models/index.model';
+
+@Component({
+  selector: 'dashboard-project-tokens',
+  templateUrl: './tokens.component.html',
+  styleUrls: ['./tokens.component.scss'],
+})
+export class ProjectTokensComponent {
+
+  public projectUid: string;
+  public tokens$: Observable<ProjectTokenModel[]>;
```

Comment:
We don't append variable names with `$` for observables. 

If this is "best practices", can you please share a link to the docs and we can raise a ticket to go through the app to improve it. For now I am keen to keep things consistent.

Reply:
I like it :+1: 
code with `$` will be more readable
https://angular.io/guide/rx-library#naming-conventions-for-observables
https://cycle.js.org/basic-examples.html#basic-examples-increment-a-counter-what-is-the-convention

Reply:
Thank you 👍 .

I raised a ticket for it #1336

Reply:
Can `$` please be removed, so we can be consistent for now and  as it will be completed as part of #1336